src/pages/EventAdministration/index.tsx

```diff
@@ -97,6 +98,7 @@ const EventAdministration = () => {
       />
       <div className={classes.root}>
         <div className={classes.content}>
+          {event && event.closed && <Alert severity='warning'>Dette arrangementet er stengt.</Alert>}
```

Comment:
Veldig smart, hadde ikke tenkt på dette! Ville kanskje lagt til noe mer margin under den slik at ikke "Endre arrangement kommer så nær
```suggestion
          {event && event.closed && <Alert severity='warning' sx={{ mb: 1 }}>Dette arrangementet er stengt.</Alert>}
```